src/Controls/src/Core/Platform/Android/Shell/SearchHandlerAppearanceTracker.cs

```diff
@@ -42,7 +33,7 @@ public SearchHandlerAppearanceTracker(IShellSearchView searchView, IShellContext
 			_control = searchView.View;
 			_searchHandler.PropertyChanged += SearchHandlerPropertyChanged;
 			_searchHandler.FocusChangeRequested += SearchHandlerFocusChangeRequested;
-			_editText = (_control as ViewGroup).GetChildrenOfType<AppCompatEditText>().FirstOrDefault();
+			_editText = (_control as ViewGroup).GetChildrenOfType<EditText>().FirstOrDefault();
```

Comment:
THIS is the important change!

Reply:
can we leverage the extension methods we have for SearchView here as an alternative to setting things here?

https://github.com/dotnet/maui/blob/main/src/Core/src/Platform/Android/SearchViewExtensions.cs#L19

Reply:
not that easy, since it's not a ISearchBar and IView , but with a bigger refactor could be done in other PR